src/realm/util/terminate.cpp

```diff
@@ -30,7 +30,7 @@
 #include <CoreFoundation/CoreFoundation.h>
 #endif
 
-#if REALM_ANDROID
+#ifdef ANDROID
```

Comment:
This should not be changed. Core is moving towards always defined macros. I.e. `REALM_ANDROID` is always `1` or `0`. We have added compiler warnings for use of undefined macros to help catch bugs from spelling mistakes like this. `#ifdef ANDR0ID` makes no warning.